platform.md

```diff
@@ -519,12 +519,22 @@ Usage:
       - All run-image config values SHALL be preserved unless this conflict with another requirement
     - MUST contain all buildpack-provided launch layers as determined by the [Buildpack Interface Specfication](buildpack.md)
     - MUST contain one or more app layers as determined by the [Buildpack Interface Specfication](buildpack.md)
-    - MUST contain a layer that includes `<launcher>`
+    - MUST contain a launcher layer containing:
```

Comment:
Can this be multiple layers, so that the launcher can be efficiently de-duplicated?